pkg/services/internal/bhs/service_test.go

```diff
@@ -155,3 +156,79 @@ func TestBlockHeadersService_ErrorCase(t *testing.T) {
 		})
 	}
 }
+
+func TestBlockHeadersService_IsValidRootForHeight(t *testing.T) {
+	const (
+		height = uint32(900_123)
+		root   = "abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789"
+	)
+
+	validRoot, err := chainhash.NewHashFromHex(root)
+	require.NoError(t, err)
+
+	type setupFn func(fix testservices.BHSFixture)
+
+	tests := []struct {
+		name      string
+		setup     setupFn
```

Comment:
Why not inline the setupFn?

```suggestion
		setup func(fix testservices.BHSFixture)
```

Reply:
I don't see any changes regarding this. And also you haven't answer my comment.

Reply:
must missed this .. probably i did other change and this got mark as outdated ... now is inline